test/broadcasting.jl

```diff
@@ -80,4 +80,11 @@ using Test
             @test names(total) == (:foo, :bar)
         end
     end
+
+    @testset "https://github.com/invenia/NamedDims.jl/issues/8#issuecomment-490124369" begin
```

Comment:
can we have a better name for this? 
and then include the link as a comment?

Reply:
I honestly can;t think of a better one.
It could be called `Regression test for #8b` ?